src/store/getters/state-getters.ts

```diff
@@ -757,8 +735,8 @@ export const areProvisionsRemoved = (state: StateIF): boolean => {
 }
 
 /** The original resolution dates. */
-export const getPreviousResolutionDates = (state: StateIF): ResolutionsIF[] => {
-  return state.stateModel.shareStructureStep.previousResolutionDates
+export const getOriginalResolutions = (state: StateIF): ResolutionsIF[] => {
+  return getEntitySnapshot(state)?.resolutions
```

Comment:
Since we no longer specially store the previous resolution dates, we can simply return the resolutions from the entity snapshot.